src/index.js

```diff
@@ -2,6 +2,9 @@ import * as themes from './themes';
 import * as numbers from './util/numbers';
 import * as currency from './util/currency';
 
+// FIXME: these exports don't work unless we introduce a babel
+//        transform. This is future ES syntax.
+
```

Comment:
Does this refer to `export { default as ... } from '...'`?

Reply:
Yup.